libs/ui/.storybook/manager-head.html

```diff
@@ -0,0 +1 @@
+<link rel="shortcut icon" href="/favicon.ico" />
```

Comment:
It's a minute problem for another day, but this favicon is super old. We should replace it across the apps.

Reply:
Ah, good to know! Makes me think I can eventually move a lot of the base html scaffolding into AppBase as well (or at least some common html template), then stuff like the favicon can just be updated in one place

Reply:
Relatedly: https://github.com/votingworks/vxsuite/issues/837.

Reply:
Ohhh, nice - I have thoughts 😁